project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.17
+sbt.version=1.2.1
```

Comment:
@lihaoyi can we scrap JDK 7 support? Upgrading the build to use 2.12 results in errors on JDK 7

```
java.lang.UnsupportedClassVersionError: scala/Option : Unsupported major.minor version 52.0
```